src/utils/client.js

```diff
@@ -216,9 +217,26 @@ export default class Client {
       additionalValues: initialValues,
       override: true
     });
+    this._loadFonts(result);
     return [steps, result];
   }
 
+  _loadFonts(res) {
+    // Load default fonts
+    if (res.fonts.length) WebFont.load({ google: { families: res.fonts } });
+    // Load user-uploaded fonts
+    Object.entries(res.uploaded_fonts).forEach((family, fontStyles) => {
+      fontStyles.forEach((fontStyle) =>
```

Comment:
could destructure fontStyle to `{source, style, weight}`

Reply:
ooh good idea!